code/datums/skills/_skill.dm

```diff
@@ -0,0 +1,6 @@
+/datum/skill
```

Comment:
Is there a particular reason each skill would be it's own datum vs. 1 skill holding datum? Idrk the processing impact but I would think it would be easier to just hold all the skills in one rather than maintaining a list of skill datums that have very little information on them individually

Reply:
they're singletons so it shouldn't matter at all anyways. The reasoning for them being datums beyond that is that skills have unique effects.  For example for mining speed modifiers make sense, for surgery it might be success rates.

Reply:
based